### DISCUSSION TITLE
Using external i18n storage for header localization

### DISCUSSION THREAD

**Discussion by nyurik** [frictionlessdata/frictionlessdata.io#865] [General]
When handling thousands of independent datasets, each describing weather, it would make sense to NOT store the translation of each column's title. I posted an idea about using a translation dictionary as a lookup source for headers and localized data as well. See https://phabricator.wikimedia.org/T134835#2906059 for details.

Question: How would we want to reference that external dictionary in the spec?

**Comment by augusto-herrmann** [comment]
I think something like this would make sense, especially in cases where:

* some column names might be the same in every language or not translatable
* l10n is in progress and you would still like people to have access to a partial translation, but show the yet untranslated labels where unavailable
